app/Console/Commands/ImageProcessing/ExtractColourPalette.php

```diff
@@ -59,8 +60,14 @@ public function handle(): int
 			}
 
 			foreach ($photos as $photo) {
-				$this->line(sprintf('Extracting Color Palette for %s [%s].', $photo->title, $photo->id));
-				ExtractColoursJob::dispatchSync($photo);
+				try {
+					$this->line(sprintf('Extracting Color Palette for %s [%s].', $photo->title, $photo->id));
+					ExtractColoursJob::dispatchSync($photo);
```

Comment:
Do we want to check `Configs::getValueAsBool('use_job_queues')` here too?

Reply:
and have it determined directly by the `.env` config